### PR TITLE
Use edeliver to deploy

### PR DESCRIPTION
**Pull request by pkoch**
Prod is already using this.

### CODE REVIEW

**Comment by nunopolonia** [comment]
[[Change deploy system] Remove gatling from the API](https://trello.com/c/pHW2uPBm/112-change-deploy-system-remove-gatling-from-the-api)

File: rel/config.exs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+end
+
+use Mix.Releases.Config,
+    default_release: :default,
+    default_environment: Mix.env()
+
+environment :dev do
+  # If you are running Phoenix, you should make sure that
+  # server: true is set and the code reloader is disabled,
+  # even in dev mode.
```
Comment (nunopolonia) [inline rel/config.exs]: We have code reloader enabled in dev mode.
I'm guessing we don't care a lot about releases in dev mode, but I thought I should point it out anyway

Comment (pkoch) [reply rel/config.exs]: Yeah, we don't want to make dev releases. I'd be down to take this out altogether.

Thoughts?

**Comment by pkoch** [comment]
The current commit only took out some unneeded configs, and put some instructions in the README.

Merging!

**Pull request merged**